op-devstack/sysgo/l2_cl_supernode.go

```diff
@@ -0,0 +1,327 @@
+				PprofConfig:   oppprof.CLIConfig{ListenEnabled: false},
+			}
+
+			// Build virtual node config for this chain (op-node config)
+			_, jwtSecret := orch.writeDefaultJWT()
+			interopCfg := &interop.Config{}
+			nodeCfg := &opnodecfg.Config{
+				L1: &opnodecfg.L1EndpointConfig{
+					L1NodeAddr:       l1EL.UserRPC(),
+					L1TrustRPC:       false,
+					L1RPCKind:        sources.RPCKindDebugGeth,
+					RateLimit:        0,
+					BatchSize:        20,
+					HttpPollInterval: 100 * time.Millisecond,
+					MaxConcurrency:   10,
+					CacheSize:        0,
+				},
+				L2: &opnodecfg.L2EndpointConfig{
+					L2EngineAddr:      l2EL.EngineRPC(),
+					L2EngineJWTSecret: jwtSecret,
+				},
+				Beacon:                      &opnodecfg.L1BeaconEndpointConfig{BeaconAddr: l1CL.beaconHTTPAddr},
+				Driver:                      driver.Config{SequencerEnabled: cfg.IsSequencer, SequencerConfDepth: 2},
+				Rollup:                      *l2Net.rollupCfg,
+				DependencySet:               nil,
+				P2PSigner:                   nil,
+				RPC:                         oprpc.CLIConfig{ListenAddr: "127.0.0.1", ListenPort: 0, EnableAdmin: true},
+				InteropConfig:               interopCfg,
+				P2P:                         nil,
+				L1EpochPollInterval:         2 * time.Second,
+				RuntimeConfigReloadInterval: 0,
+				Tracer:                      nil,
+				Sync: nodeSync.Config{SyncMode: func() nodeSync.Mode {
+					if cfg.IsSequencer {
+						return cfg.SequencerSyncMode
+					}
+					return cfg.VerifierSyncMode
+				}()},
+				ConfigPersistence:               opnodecfg.DisabledConfigPersistence{},
+				Metrics:                         opmetrics.CLIConfig{},
+				Pprof:                           oppprof.CLIConfig{},
+				SafeDBPath:                      cfg.SafeDBPath,
+				RollupHalt:                      "",
+				Cancel:                          nil,
+				ConductorEnabled:                false,
+				ConductorRpc:                    nil,
+				ConductorRpcTimeout:             0,
+				AltDA:                           altda.CLIConfig{},
+				IgnoreMissingPectraBlobSchedule: false,
+				ExperimentalOPStackAPI:          true,
+			}
+			_ = nodeCfg // ensure used below
+
+			n := &SuperNode{
+				id:     l2CLID,
+				p:      p,
+				logger: logger,
+				el:     &l2ELID,
+				snCfg:  snCfg,
+				vnCfgs: map[sntypes.ChainID]*opnodecfg.Config{chainID: nodeCfg},
+			}
+			require.True(orch.l2CLs.SetIfMissing(l2CLID, n), fmt.Sprintf("must not already exist: %s", l2CLID))
+			aggregator.addChain(n, p, snCfg, chainID, nodeCfg, logger)
+		}),
+		stack.Finally(func(orch *Orchestrator) {
+			aggregator.start(orch.P())
+			orch.P().Cleanup(func() { aggregator.stop(orch.P()) })
+		}),
+	)
+}
+
+// findFreePort returns an available localhost TCP port.
+func findFreePort(p devtest.P) int {
+	l, err := net.Listen("tcp", "127.0.0.1:0")
+	p.Require().NoError(err)
+	defer l.Close()
+	return l.Addr().(*net.TCPAddr).Port
+}
+
+// no extra helpers
```

Comment:
I find this comment confusing. What is it telling us?

Reply:
Just cruft from an old edit, will be removed.